### ISSUE TITLE
Where can I get surface normal images of blender dataset.

### ISSUE THREAD

**Issue by K-nowing** [half-potato/nmf#14]
Hi, where can I get surface normal images of blender dataset.
The value of the surface normal image in the data accessible from [the official github page](https://github.com/bmild/nerf) seems to be an inaccurate value.
Thank you so much!

<img width="332" alt="image" src="https://github.com/half-potato/nmf/assets/78612464/ab5fdcc7-53fa-4f06-bb54-5599986245a3">

The predicted normal from the paper.

<img width="332" alt="image" src="https://github.com/half-potato/nmf/assets/78612464/7ba5d992-f40b-45de-b746-42da31b3fede">

The GT normal from the data.

**Comment by half-potato** [comment]
I believe these need to be mapped from [-1, 1] to [0, 1] by dividing by 2 and adding 0.5.

**Comment by K-nowing** [comment]
The predicted normal image is already mapped from [-1, 1] to [0, 1].

**Comment by half-potato** [comment]
Check the ground truth.

**Comment by K-nowing** [comment]
The ground truth is a png file, so it ranges from 0 to 255. Even if I change the range from [0, 255] to [-1, 1], it does not work. Am I missing something? Thank you for your reply!

**Comment by K-nowing** [comment]
+) From this [line](https://github.com/half-potato/nmf/blob/489f53da177f471da0841bdb2b99bef6ddbeec88/dataLoader/blender.py#L152C62-L152C72), it means that the suffix of the normal image file is "_normal.png". However the suffix of the normal image from [the official github page](https://github.com/bmild/nerf)  is "_normal_0000.png".

**Comment by half-potato** [comment]
Did you get this working?

**Comment by K-nowing** [comment]
No, it doesn't work... The GT normal image that I have seems to be different from the one you have.

**Comment by half-potato** [comment]
This can happen if the normals are clipped before being exported, so you don't have the correct range on the normals. In this case, the ground truth normals have the wrong range.
